### PR TITLE
Update it.nsi

### PR DESCRIPTION
**Pull request by dzpex**

### CODE REVIEW

File: src/res/translation/wininstaller/it.nsi [hunk 1]
```diff
@@ -10,28 +10,28 @@ LangString RUNNING_APP_MSG ${LANG_ITALIAN} \
...
 LangString ASIO_DRIVER_EXPLAIN ${LANG_ITALIAN} \
-     "Jamulus needs an ASIO driver to provide low latency audio. More information:"
+     "Jamulus ha bisogno dei driver ASIO al fine di garantire una bassa latenza audio. Maggiori Informazioni:"
 
 LangString ASIO_DRIVER_MORE_INFO ${LANG_ITALIAN} \
-     "More information about ASIO on jamulus.io"
+     "Maggiori informazioni sui driver ASIO disponibili su jamulus.io"
 
 LangString ASIO_DRIVER_MORE_INFO_URL ${LANG_ITALIAN} \
      "https://jamulus.io/wiki/Installation-for-Windows#asio"
```
Comment (ann0see) [inline src/res/translation/wininstaller/it.nsi:34]: I think you could link the italian page here too (if it's translated)

**Review by ann0see — CHANGES_REQUESTED** [review]
- [x] Translator listed in the `src/util.cpp` https://github.com/jamulussoftware/jamulus/blob/387e1125315d1c66be0ae2f6569fdcce99e1974b/src/util.cpp#L503
- [] Looks consistent
- [x] Passes `tools/check-wininstaller-translations.sh` Yes. But it shouldn't since Italian is commented (@hoffie)

Great! Thank you. I think you'll have to uncomment the italian include in installerlng.nsi too.

**Comment by dzpex** [comment]
ok i make the mods in the installerlng.nsi

**Comment by ann0see** [comment]
Great! Once the CI is finished, you can test the new installer by downloading the artifacts from the checks page in this PR (will probably take ~ 1h).

**Review by ann0see — APPROVED** [review]
The installer still should be checked. But all in all this looks good.

**Review by hoffie — APPROVED** [review]
Looks good time. Also checked the installer.

@ann0see I will send a PR for the check script.

**Pull request merged**